packages/wdio-webdriver-mock-service/src/WebDriverMock.ts

```diff
@@ -56,17 +60,21 @@ export default class WebDriverMock {
         }
     }
 
-    get(obj, commandName) {
-        const { method, endpoint, commandData } = protocolFlattened.get(commandName)
+    get(obj: any, commandName: string) {
+
+        const protocolFltattenedValues = protocolFlattened.get(commandName)
+        const method = protocolFltattenedValues!.method
+        const endpoint = protocolFltattenedValues!.endpoint
+        const commandData = protocolFltattenedValues!.commandData
```

Comment:
Why do we ignore the type check here?
```suggestion
        const method = protocolFltattenedValues.method
        const endpoint = protocolFltattenedValues.endpoint
        const commandData = protocolFltattenedValues.commandData
```

Reply:
Added type for "{ method: string, endpoint: string, commandData: WDIOProtocols.CommandEndpoint }"